### ISSUE TITLE
Make project group and project cards more visually pleasing

### ISSUE THREAD

**Issue by e-lo** [ZephyrTransport/zephyr-directory#4]
Right now they are gray blobs.  What should we do?  Hmm.

**Comment by billyc** [comment]
Even just a splash of color on each would wake things up a bit. I'm sick of flat icons... maybe some fancy background fullbleed images? hmm.  We need an official Zephyr colorscheme.

**Comment by e-lo** [comment]
Agreed. I'll add "create color scheme" as a TODO.  I sorta created a turquoise/gray scheme on main website [zephyrtransport.org](zephyrtransport.org) but am happy to modify.

Please add examples of websites you like w.r.t. "fancy background fullbleed image" and "not flat icons" :-)
